Documentation/DataSources/DataSources.md

```diff
@@ -10,13 +10,15 @@ Workbooks support these data sources:
 * [Azure Resource Graph](#azure-resource-graph-arg)
 * [Azure Resource Manager](#azure-resource-manager)
 * [Azure Data Explorer](#azure-data-explorer)
-* [Workload Health](#workload-health)
 * [Azure Resource Health](#azure-resource-health)
 * [JSON](#json)
-* [Alerts](#alerts-preview)
 * [Custom Endpoint](#custom-endpoint)
 * Custom Provider
 
+Deprecated (Previously available):
+* [Alerts](#alerts-preview)
+* [Workload Health](#workload-health)
```

Comment:
deprecated or removed?

Reply:
technically, both still exist in code. alerts works but can't be added to *new* workbooks, (works in ones where it already is used), workload health doesn't work and *will be* removed, but still also exists in code and will until it is removed and works through deployments.